src/CircleButton/CircleButton.js

```diff
@@ -22,5 +22,5 @@ NavCircleButton.defaultProps ={
 NavCircleButton.propTypes = {
   className: PropTypes.string.isRequired,
   children: PropTypes.node.isRequired,
-  tag: PropTypes.func.isRequired
+  tag: any
```

Comment:
Changed to allow any types since it wasn't being used consistently across the application to strongly type it. Code smell, could be something to improve if ever going into production

- [Function usage](https://github.com/Ejesalva840/note-taking/blob/2748142d384790a66a6c24e14404e4c7f3d33750/src/NoteListMain/NoteListMain.js#L38) 

- [Button usage](https://github.com/Ejesalva840/note-taking/blob/2748142d384790a66a6c24e14404e4c7f3d33750/src/NotePageNav/NotePageNav.js#L28)